src/app/services/account.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+from rest_framework.request import Request
+from app.domain.models.account import Account
+from app.domain.repositories.account import AccountRepository
+
+
+class AccountService:
+    def __init__(self, account_repository: AccountRepository):
+        self.account_repository = account_repository
+        self.model = Account
+
```

Comment:
Algum motivo em específico para preferir a implementação com o `__init__` ao invés de utilizar atributos de classe?

> fazer dessa forma facilita a criação do Service sem precisar importar o Repository.

Por exemplo, em algum momento vc vê que seria útil poder injetar OUTRO repository que não seja o AccountRepository?

```suggestion
class AccountService:
    repository = AccountRepository()
    model = Account
```